AppSettingsManagerUI/src/app/login/login.component.ts

```diff
@@ -0,0 +1,53 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { FormGroup, FormControl, Validators } from '@angular/forms';
+import { AuthService } from '../services/auth/auth.service';
+import { BffService } from '../services/bff/bff.service';
+
+@Component({
+  selector: 'app-login',
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.scss']
+})
+export class LoginComponent implements OnInit {
+  loginForm: FormGroup;
+  signUpForm: FormGroup;
+  constructor(private authService: AuthService, private router: Router, private bffService: BffService) {
+    this.loginForm = new FormGroup({
+      usernameLogin: new FormControl('', [Validators.required, Validators.minLength(4)]),
+      passwordLogin: new FormControl('', [Validators.required, Validators.minLength(4)])
+    });
+    this.signUpForm = new FormGroup({
+      username: new FormControl('', [Validators.required, Validators.minLength(4)]),
+      password: new FormControl('', [Validators.required, Validators.minLength(4)]),
+      email: new FormControl('', [Validators.required])
+    });
+  }
+
```

Comment:
In the future, please move validation rules to a separate file as it makes them reusable across different components. I would also recommend looking into RxJS for data flow management as it will help make the code more reactive.